show/system_health.py

```diff
@@ -159,3 +166,76 @@ def sysready_status_detail():
         clicommon.run_command(cmd, display_cmd=False)
     except Exception as e:
         click.echo("Exception: {}".format(str(e)))
+
+
+def show_module_state(module_name):
+    chassis_state_db = SonicV2Connector(host=CHASSIS_SERVER, port=CHASSIS_SERVER_PORT)
+    chassis_state_db.connect(chassis_state_db.CHASSIS_STATE_DB)
+    key = 'DPU_STATE|'
```

Comment:
@rameshraghupathy I don't see any check here that prevents this function being called on non-smartswitch platform

Reply:
This won't be executed on regular switch
def dpu(module_name):
    if not is_smartswitch():
        return
    """Show system-health dpu information"""
    show_module_state(module_name)
@prgeor